packages/mcp-server-supabase/src/server.test.ts

```diff
@@ -377,6 +378,41 @@ describe('tools', () => {
     });
   });
 
+  test('try creating a project in read-only mode', async () => {
```

Comment:
Test names should include the expected outcome for non-happy paths:

```suggestion
  test('create project in read-only mode throws an error', async () => {
```